packages/ppx_getenv/ppx_getenv.2.1/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
```

Comment:
weird.. @NathanReb is that expected? Maybe it's my branch that has an issue, I'm using https://github.com/kit-ty-kate/dune-release/tree/custom (with a few of my recent PRs + a fix for it to compile with opam 2.1)

Reply:
oh, mh unless dune-release is doing something with the comments I'm guessing this is an effect of the upcoming https://github.com/ocaml/opam/pull/4302 that I'm using locally